src/FFmpegWriter.cpp

```diff
@@ -1231,11 +1220,46 @@ AVStream *FFmpegWriter::add_video_stream() {
 		// Here should be the setting for low fixed bitrate
 		// Defaults are used because mpeg2 otherwise had problems
 	} else {
-		// Check if codec supports crf
+		// Check if codec supports crf or qp
 		switch (c->codec_id) {
 #if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(55, 39, 101)
 #if (LIBAVCODEC_VERSION_MAJOR >= 58)
 			case AV_CODEC_ID_AV1 :
+						// TODO: Set `crf` or `qp` according to bitrate, as bitrate is not supported by these encoders yet.
```

Comment:
NOTE: I think the indention on this block seem too large, perhaps from copy/pasting.

Reply:
Yes, that is correct. I'm pretty sure that was copy-n-paste